File: python/native_issue_action.py
from client import create_gateway_client
import compassiot.native.v1.native_pb2 as native


def main():
    client = create_gateway_client()

    request = native.IssueActionRequest(
        vin="",
        lock=native.SetLockCommand(
            locked=True
        )
    )

    # Returns empty, check using NativeGetLatestPoint
    client.NativeIssueAction(request)

     
if __name__ == "__main__":
    main()

File: typescript/src/native_issue_action.ts
import * as native from "@buf/compassiot_api.bufbuild_es/compassiot/native/v1/native_pb"

import { createNodeClient } from "./client"

const client = createNodeClient()

const request = new native.IssueActionRequest({
  vin: "",
  command: {
    case: "lock",
    value: new native.SetLockCommand({
      locked: true
    })
  }
})

// Returns empty
await client.nativeIssueAction(request)
